src/views/GeneralReception.vue

```diff
@@ -272,6 +272,7 @@ export default {
       attributes: [], // attributes is used to store the attributes that have been found in sequencescape
     },
     printerName: '',
+    debounceTimer: 1000, // debounce timer for barcode deletion
```

Comment:
I think this should be set to null by default since its not the time its the timer itself. I think thats my mistake when we went over it.

Reply:
@BenTopping 
No, you helped a lot. It's bit stupid of me that I didn't realise that it's a timerId returned by setTimeout and not a timer value.
Updated to null now 👍